src/test/java/seedu/address/logic/commands/FindCommandTest.java

```diff
@@ -55,29 +63,65 @@ public void equals() {
     }
 
     @Test
-    public void execute_zeroKeywords_noPersonFound() {
-        String expectedMessage = String.format(MESSAGE_PERSONS_LISTED_OVERVIEW, 0);
-        NameContainsKeywordsPredicate predicate = preparePredicate(" ");
+    public void execute_oneField_multiplePersonsFound() {
+        PersonMatchesKeywordsPredicate predicate = new PersonMatchesKeywordsPredicateBuilder()
+                .withName(KEYWORD_MATCHING_MEIER).build();
+        String expectedMessage = String.format(MESSAGE_FIND_RESULTS, 2, predicate);
         FindCommand command = new FindCommand(predicate);
         expectedModel.updateFilteredPersonList(predicate);
         assertCommandSuccess(command, model, expectedMessage, expectedModel);
-        assertEquals(Collections.emptyList(), model.getFilteredPersonList());
+        assertEquals(Arrays.asList(BENSON, DANIEL), model.getFilteredPersonList());
     }
 
     @Test
-    public void execute_multipleKeywords_multiplePersonsFound() {
-        String expectedMessage = String.format(MESSAGE_PERSONS_LISTED_OVERVIEW, 3);
-        NameContainsKeywordsPredicate predicate = preparePredicate("Kurz Elle Kunz");
+    public void execute_multipleFieldsAllCondition_multiplePersonsFound() {
+        // Default condition
+        PersonMatchesKeywordsPredicate predicate = new PersonMatchesKeywordsPredicateBuilder().withAddress("ave")
+                .withTags("friends").build();
+        String expectedMessage = String.format(MESSAGE_FIND_RESULTS, 2, predicate);
```

Comment:
Perhaps it may be better to extract out `2` as a variable `int expectedNumPersons = 2`? Same for the other tests.